Replace squash with honeybadger

cbeer opened the pull request.

cbeer committed:
Replace squash with honeybadger

atz committed:
Update solr_wrapper to get new version of solr

... else CI breaks because old solr's checksum file is unavailable.

atz committed:
Use .rspec file

Get color and consolidation of `require "rails_helper"`.

atz committed:
bundle update blacklight-spotlight

atz committed:
Update README: Portfolios is not Exhibits.

It's one thing to not have badges, but it is quite another to display
badges from a different codebase!  Or, for that matter, not to update
the content at all, from the content that wouldn't have even been
correct even in the *old* repo!

atz commented:
I think this will work after we merge my updates branch and get the new solr_wrapper.

atz committed:
Add newrelic

atz committed:
Cleanup Gemfile

dlss-capistrano sets the req'd semver of capistrano for us.
Also, you should be running rubocop, coverage tools and tests during
development.  These are not a "test env only" add-on.

atz committed:
Add slowpoke w/ 60s timeout

atz committed:
Cap file for stage

Also remove lock from deploy.rb.

atz commented:
Rebased on my branch.  Will fold the honeybadger work into it.

atz commented:
Closing, superceded by #7
